test/open_mic_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require './lib/user'
+require './lib/open_mic'
+
+class OpenMicTest < Minitest::Test
+
+  def setup
+    @sal = User.new("Sal")
+    @ali = User.new("Ali")
+    @joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    @joke_2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+    @open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+  end
+
+  def test_it_exists
+    assert_instance_of OpenMic, @open_mic
+  end
+
+  def test_open_mic_has_attributes
+    assert_equal "Comedy Works", @open_mic.location
+    assert_equal "11-20-18", @open_mic.date
+    assert_equal [], @open_mic.performers
+  end
+
+  def test_welcome_performers
+    @open_mic.welcome(@sal)
+    @open_mic.welcome(@ali)
+    assert_equal [@sal, @ali], @open_mic.performers
+  end
+
+  def test_if_jokes_have_been_repeated
+    @ali.learn(@joke_1)
+    @ali.learn(@joke_2)
+    assert_equal false, @open_mic.repeated_jokes?
+  end
```

Comment:
this is a good start on this last test, but we'll want to see the method return a true as well to be FULLY tested